api/README.md

```
@@ -0,0 +1,64 @@
+   nvm install
+   ```
+
+3. Instale as dependências do projeto
+
+   ```
+   pnpm install
+   ```
+
+### Estrutura de arquivos e pastas
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)